quark/mdk-2.0.q

```diff
@@ -788,6 +788,12 @@ namespace mdk {
         }
 
         void start_interaction() {
```

Comment:
Public API? Why underscore vs camelcase... I wish Quark could just "do the right thing" and generate idiomatic API's across languages.